### DISCUSSION TITLE
大霸王 | /posts/big-overlord/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#611] [Announcements]
过了三十岁以后我总对四十岁感到好奇，那传说中的“四十不惑”会是真的吗？当然，如果将来我能够无恙地活到四十岁，也会继续好奇传说中的“五十知天命”是不是真的呢？

https://yuanfan.rbind.io/posts/big-overlord/

**Comment by hongtaoh** [comment]
1. 我差点把晗晗认成是鹿晗。
2. 每年你放烟花估计很开心吧
3. 奶奶问年龄那段莫名有一种生动的感觉，我都能想到奶奶问问题的神情。其实我现在觉得问这些问题的大家伙们也不是关心，只是为了怕没话说觉得尴尬才问的，就跟「吃了没有」一个道理。

**Reply by earfanfan** [reply]
1. 你还真是郝糊涂啊，这差点也差得太大点了吧。
2. 现在烟花做得比以前好了，引线都很长，所以尽管胆子还是很小，但是都敢自己去点了，放烟花的乐趣比看烟花的乐趣多一点点。
3. 这个嘛，我也感觉别人问比较多是八卦或者好奇的心态，不把别人理解成恶意对自己也好。话说我昨晚遛狗时遇到小孩，居然也会聊这个话题。

小孩：你家有孩子吗？
我：没有。
小孩：（漫长的震惊）未……未婚？那你是不打算结婚生小孩了吗？
我：不是啊，只是运气不好，没遇到合适的人。
小孩：那你怎么不去相亲呢？
我：天呐，你只是一个十岁的小孩，等你长大了自然就会懂了。
小孩：其实我懂，有人追我呢blabla，blablabla。

**Reply by hongtaoh** [reply]
还小孩估计是女孩子吧。现在的孩子都早熟。

>不是啊，只是运气不好，没遇到合适的人。

我觉得这句需要很大的智慧啊。能心平气和说出来很厉害，我做不到，接受不了。

**Reply by earfanfan** [reply]
那是一个十岁的男孩子。现在不兴那种非要男孩追女孩了，也有很多女孩会主动追求自己喜欢的男孩的。

说这句话不需要什么大智慧啊，我想要的是那种，两个完整独立的人互相欣赏、尊重、支持，一起快乐地生活。按照过往去相亲得到的历史经验，我想要的是极其罕见的高标准高要求，是我自己不愿意为了生存和完成所谓人生任务而瞎凑合，这也没法怪别人呐。所以真得就是运气不好，连达到这个标准百分之二三十的都没有见到过。当然啦，我接受事实也并不代表我一点抱怨都无，哈哈，其实我有时候也会觉得这个锅应该甩到时代的头上去。

**Reply by hongtaoh** [reply]
为啥，这个跟时代有啥关系呢

**Reply by earfanfan** [reply]
额，在我接触到的世界的这一面，我觉得完整独立的人格是普通标准，结果不知道撒原因变成了罕见的高标准，懂得尊重别人本来也应该只是普通标准，结果在方方面面不平等（包括男女不平等）的世界也变成了罕见的高要求。所以我找不到对象这件事跟时代的关系是，大环境下人们可以自由选择的选择太少了，显得像我这样正常的人是人群中的极少数“异类”，如果大家全都拥有完整独立的人格并且都懂得尊重别人，那像我这样的人找对象就会变得很容易啦。
